adapter-s4r/src/main/resources/sanity-validation/e2e-nbs4r-sanity.sh

```diff
@@ -0,0 +1,137 @@
+#! /usr/local/bin/bash
```

Comment:
Let's work together to put these in native nb scenario script form or CLI script form.

Reply:
Deleted the bash scripts (as per our discussion)